src/jit/gentree.h

```diff
@@ -1332,7 +1332,7 @@ struct GenTree
 
     bool OperIsMultiRegOp() const
     {
-#if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
+#if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_) && defined(ARM_SOFTFP)
```

Comment:
Can you explain why this is only applicable to `ARM_SOFTFP`? It has uses in lsra.cpp and regset.cpp that I believe are applicable to non-SOFTFP ARM.